_includes/image.html

```diff
@@ -1,15 +1,56 @@
 {% capture native-url %}{{ include.loc }}/{{ site.lang }}/{{ include.file }}{% endcapture %}
 {% capture english-url %}{{ include.loc }}/en/{{ include.file }}{% endcapture %}
+{% capture single-url %}{{ include.loc }}/{{ include.file }}{% endcapture %}
 {% assign img-alt = include.alt | remove:'<!-- mdpo-enable-next-line -->' | strip %}
 
 {% for file in site.static_files %}
    {% if file.path == native-url %}
       {% assign native-exists = true %}
+   {% elsif file.path == english-url %}
+      {% assign english-exists = true %}
    {% endif %}
 {% endfor %}
 
-{% if native-exists == true %}
+<a href="" data-toggle="modal" data-target="#imgModal">
+{%- if native-exists == true %}
 <img src="{{ native-url }}" alt="{{ img-alt }}" width="{{ include.width }}" class="{{ include.class }}">
-{% else %}
+{%- elsif english-exists == true %}
 <img src="{{ english-url }}" alt="{{ img-alt }}" width="{{ include.width }}" class="{{ include.class }}">
-{% endif %}
+{%- else %}
+<img src="{{ single-url }}" alt="{{ img-alt }}" width="{{ include.width }}" class="{{ include.class }}">
+{%- endif %}
+</a>
+
+<script type="text/javascript">
```

Comment:
This script is included multiple times if there are multiple images. It should instead be in the JavaScript file

Reply:
Whoops. That script isn't even needed for the images.